pkg/karmadactl/cmdinit/kubernetes/deployments.go

```diff
@@ -65,7 +65,7 @@ func (i *CommandInitOption) karmadaAPIServerContainerCommand() []string {
 		fmt.Sprintf("--client-ca-file=%s/%s.crt", karmadaCertsVolumeMountPath, options.CaCertAndKeyName),
 		"--enable-admission-plugins=NodeRestriction",
 		"--enable-bootstrap-token-auth=true",
-		fmt.Sprintf("--etcd-cafile=%s/%s.crt", karmadaCertsVolumeMountPath, options.CaCertAndKeyName),
+		fmt.Sprintf("--etcd-cafile=%s/%s.crt", karmadaCertsVolumeMountPath, options.EtcdCaCertAndKeyName),
```

Comment:
Should we modify in 
`https://github.com/karmada-io/karmada/blob/master/pkg/karmadactl/addons/search/manifests.go#:~:text=%2D%20%2D%2Detcd%2Dservers%3D%7B%7B%20.ETCDSevers,pki/karmada.crt` similarly

Reply:
Thanks for reminding, fixed.